### PR TITLE
Two little tests

### PR DESCRIPTION
**Pull request by kshyatt**
For `type_create` and for `Testall!`

### CODE REVIEW

**Comment by kshyatt** [comment]
Can someone help me out with the Windows failure?

**Comment by eschnett** [comment]
The Appveyor are still under development; please ignore the failure for the moment.

File: test/test_sendrecv.jl [hunk 1]
```diff
@@ -30,6 +30,10 @@ stats = MPI.Waitall!([sreq, rreq])
 @test MPI.Get_tag(stats[2]) == src+32
 @test isapprox(norm(recv_mesg-recv_mesg_expected), 0.0)
 
+(done,stats) = MPI.Testall!([sreq, rreq])
+@test done == true
```
Comment (eschnett) [inline test/test_sendrecv.jl]: `@test done` suffices, no need to compare to `true`.

**Comment by davidanthoff** [comment]
Actually, the julia 0.4 appveyor test should not fail. There are some constants missing, you can easily add them like in #139 (actually I think one of the missing constants is added in #139, no probably best to merge that first and then rebase on master).

**Comment by kshyatt** [comment]
@davidanthoff I can add `MPI_WAITANY` and `MPI_TESTANY` (and whatever else is missing) in this PR as well, and owe you credit and 🍻 ?

**Comment by davidanthoff** [comment]
@kshyatt Sounds good!

**Comment by kshyatt** [comment]
Derp, I updated the wrong PR. Sorry!

**Comment by kshyatt** [comment]
At least one of the Windows tests is passing - this look ok, @andreasnoack and @davidanthoff ?

**Comment by davidanthoff** [comment]
Looks good to me. One of the julia 0.4 tests also didn't finish, but that looks like a timeout on appveyor's side that has nothing to do with this PR, so I would ignore it.

**Pull request merged**